tests/subsys/logging/log_stack/src/maincxx.cxx

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2019 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <sys/util.h>
+#include <string.h>
```

Comment:
AFAICT these #includes should be removed or moved into test.inc. And the same for main.c.